test/functional/interface_rest.py

```
@@ -57,14 +58,21 @@ def set_test_params(self):
         self.supports_cli = False
 
-    def test_rest_request(self, uri, http_method='GET', req_type=ReqType.JSON, body='', status=200, ret_type=RetType.JSON):
+    def test_rest_request(
+            self,
+            uri: str,
+            http_method: str = 'GET',
+            req_type: ReqType = ReqType.JSON,
+            body: str = '',
+            status: int = 200,
+            ret_type: RetType = RetType.JSON,
+            query_params: typing.Dict[str, typing.Any] = None,
```

Ruff (0.12.2)

Use `dict` instead of `typing.Dict` for type annotation

Replace with `dict`

(UP006)